src/higlass/tilesets.py

```diff
@@ -74,10 +75,14 @@ def remote(uid: str, server: str = "https://higlass.io/api/v1", **kwargs):
 def hash_absolute_filepath_as_default_uid(
     fn: typing.Callable[[str, str], LocalTileset]
```

Comment:
This change would change the type annotation here and in the functions below.

Reply:
True. Updated.